Lesson7.ipynb

```diff
@@ -29,6 +33,551 @@
+      "text/plain": [
+       "<matplotlib.figure.Figure at 0x1184161d0>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "#Step 6\n",
+    "barchartdf.plot.bar()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Homework:\n",
+    "1. Create vertical bar chart with the team_name category for Saturday only.\n",
+    "2. Create horizontal bar chart using [barh() function](http://pandas.pydata.org/pandas-docs/version/0.22/generated/pandas.DataFrame.plot.barh.html)."
+   ]
```

Comment:
We usually add an empty code cell per homework with the requirement as a comment.